packages/vm/src/cache.rs

```diff
@@ -53,6 +53,14 @@ pub struct Metrics {
     pub size_memory_cache: usize,
 }
 
+#[derive(Debug, Clone)]
+pub struct HeavyMetrics {
+    // It is *intentional* that this is only a vector
+    // We don't need a potentially expensive hashing algorithm here
+    // The checksums are sourced from a hashmap already, ensuring uniqueness of the checksums
+    pub hits_per_pinned_contract: Vec<(Checksum, u32)>,
+}
```

Comment:
Can we add element size here as well? This we already have in the cache but only expose the sum in the normal stats. This allows creating overviews of the elements in there, their cost (size) and value (hits).

Reply:
Sure, seems reasonable

Reply:
Done. The field was renamed to `per_module`. Key is still the checksum, value is a struct containing the hits and the size estimate.